news/2024/2024-04-07-vocaloid-mapping-contest-2-results.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: "Vocaloid Mapping Contest 2 Results"
+date: 2024-04-07 20:00:00 +0000
+---
+
+We have all been eagerly awaiting the results of the contest themed around the incredible music of [iroha(sasaki)](https://osu.ppy.sh/beatmaps/artists/399). Well, worry no longer, for they are here!
+
+![](/wiki/Contests/VMC/2/img/banner.jpg)
+
+The [**Vocaloid Mapping Contest** (***VMC***)](/wiki/Contests/VMC) features vocasynth music that hasn't received the appreciation it deserves. The task that mappers were challenged with was to create the best map using [one of the 5 songs provided](https://osu.ppy.sh/home/news/2024-01-28-vocaloid-mapping-contest-2#songs). Now, normally that is challenging on it's own, but it's not every day that your maps get judged by a VOCALOID mapping legend, namely **[NatsumeRin](https://osu.ppy.sh/users/151679)**!
+
+Below we will go through the results for each of the modes, so feel free to skip to the one you're interested in!
+
+- [osu! results](#osu!-results)
+- [osu!taiko results](#osu!taiko-results)
+- [osu!catch results](#osu!catch-results)
+- [osu!mania results](#osu!mania-results)
+- [Judging criteria](/wiki/Contests/VMC/2#judging-criteria)
+
+### osu! results
+
+This contest's submissions were immediately sent to the eyes of our discerning [judges](/wiki/Contests/VMC/2#judges) after the mapping phase. Despite the relatively short mapping phase, the participants did a great job, and their passion for the music is clear.
+
+[See the full results here.](https://mappersguild.com/contests/results?contest=65d91885b2923e167f79fd44)
+
+#### Summary
+
+| Placement | User | Anonymised beatmap | Badge prize |
+| :-: | :-- | :-- | :-- |
+| 1st | [Nuolong](https://osu.ppy.sh/users/4732455) | world language | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
```

Comment:
![image](https://github.com/ppy/osu-wiki/assets/36758269/5394c9f6-2f8c-4dfd-a959-fe3227719d4f)

this badge is a bit too big for the table cell, should use the 1x version instead

Reply:
never.. mind?

![image](https://github.com/ppy/osu-wiki/assets/36758269/44aaa319-38df-47b8-88ec-39eab3c5c63d)

Reply:
wala applied